StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...

With the current version on `master` (1.0.6.9999) I get the following error logged to the R console:
`ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...`

> If you would like to provide a logging implementation in order not to see this message, do the following:
> 
>     * find the installation location(s) of packages using `.libPaths()`
> 
>     * in one of the locations, you will find an _XLConnect_ directory
> 
>     * download [log4j core 2.19](https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.19.0/log4j-core-2.19.0.jar) - this may be out of date! see https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core for all versions.
> 
>     * Put the downloaded jar in the _/XLConnect/java_ directory.
> 
>     * Restart your R session and reload XLConnect.
> 
> 
> Note that this is not necessary to use XLConnect.

We tried this and it broke our script. Instead of the error going away our script failed with this error:

```
ng.log4j.core.selector.ClassLoaderContextSelector.getContext(ClassLoaderContextSelector.java:123)
	at org.apache.logging.log4j.core.selector.ClassLoaderContextSelector.getContext(ClassLoaderContextSelector.java:117)
	at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log4jContextFactory.java:150)
	at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log4jContextFactory.java:47)
	at org.apache.logging.log4j.LogManager.getContext(LogManager.java:196)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:599)
	at org.apache.poi.openxml4j.util.ZipSecureFile.(ZipSecureFile.java:37)
	at com.miraisolutions.xlconnect.Workbook.(Workbook.java:66)
	at com.miraisolutions.xlconnect.integration.r.RWorkbookWrapper.(RWorkbookWrapper.java:52)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at RJavaTools.newInstance(RJavaTools.java:358)
Caused by: java.lang.ClassNotFoundException
	at RJavaClassLoader.findClass(RJavaClassLoader.java:397)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 33 more
Error: NoClassDefFoundError (Java): Could not initialize class org.apache.logging.log4j.core.impl.ThreadContextDataInjector
Execution halted
```

--- a/.github/workflows/ci-tests.yml
+++ b/.github/workflows/ci-tests.yml
@@ -259,9 +259,9 @@ jobs:
           set -eux -o pipefail
           XLC_JAR_PATH=$(Rscript -e "cat(.libPaths()[1])")/XLConnect/java
           rm -f "$XLC_JAR_PATH"/XLConnect*
-          find "$XLC_JAR_PATH" -maxdepth 1 -type f |\
+          find "$XLC_JAR_PATH" -maxdepth 1 -type f | grep ".jar"|\
           xargs -l sha512sum | sort | awk '{print $1}' > effective_hashes
-          find /home/runner/work/xlconnect/xlconnect/target/dependency/ -maxdepth 1 -type f |\
+          find /home/runner/work/xlconnect/xlconnect/target/dependency/ -maxdepth 1 -type f | grep ".jar"|\
           grep -v -E "poi-ooxml-lite|junit|hamcrest" |\
           xargs -l sha512sum | sort | awk '{print $1}' > expected_hashes
           cmp expected_hashes effective_hashes ||\

--- a/.github/workflows/ci-tests.yml
+++ b/.github/workflows/ci-tests.yml
@@ -259,9 +259,9 @@ jobs:
           set -eux -o pipefail
           XLC_JAR_PATH=$(Rscript -e "cat(.libPaths()[1])")/XLConnect/java
           rm -f "$XLC_JAR_PATH"/XLConnect*
-          find "$XLC_JAR_PATH" -maxdepth 1 -type f | grep ".jar"|\
+          find "$XLC_JAR_PATH" -maxdepth 1 -type f -name "*.jar"|\
           xargs -l sha512sum | sort | awk '{print $1}' > effective_hashes
-          find /home/runner/work/xlconnect/xlconnect/target/dependency/ -maxdepth 1 -type f | grep ".jar"|\
+          find /home/runner/work/xlconnect/xlconnect/target/dependency/ -maxdepth 1 -type f -name "*.jar"|\
           grep -v -E "poi-ooxml-lite|junit|hamcrest" |\
           xargs -l sha512sum | sort | awk '{print $1}' > expected_hashes
           cmp expected_hashes effective_hashes ||\

--- a/NEWS
+++ b/NEWS
@@ -1,5 +1,7 @@
 XLConnect News
 --------------
+1.0.8 tbd
+ * avoid log4j2 error on first use (#172) 
 
 1.0.7 2023-01-19
   * Fix issue with loadWorkbook (#181)

--- a/inst/java/log4j2.system.properties
+++ b/inst/java/log4j2.system.properties
@@ -1,0 +1,1 @@
+log4j2.loggerContextFactory=org.apache.logging.log4j.simple.SimpleLoggerContextFactory

--- a/inst/java/log4j2.system.properties
+++ b/inst/java/log4j2.system.properties
@@ -1,1 +1,2 @@
 log4j2.loggerContextFactory=org.apache.logging.log4j.simple.SimpleLoggerContextFactory
+rootLogger.level=OFF

